### PR TITLE
[BIT] 引擎 accuracy 回测修复 torch error：rfft2 flip frac thresholded_relu

### PR DESCRIPTION
**Pull request by Cutelemon6**
## paddle.fft.rfft2

更新 mapping.json 中的规则，原来是默认生成的，axes == None 时需要指定 torch 的 dim 为 (-2, -1)，并且由于 axes 是可选参数，需要使用 locals().get('axes') 来判断。

```json
"torch_kwargs": {
    "dim": "(-2, -1) if locals().get('axes') is None else axes"
}
```

配置均通过，将 pending 中的移入 accuracy_8.txt

## paddle.flip

需要将输入的 int 包装成 list

```json
"torch_kwargs": {
    "dims": "[axis] if isinstance(axis, int) else axis"
}, 
```

配置均通过，将 pending 中的移入 accuracy_8.txt

## paddle.frac

api 功能是获取 tensor 中每个元素的小数，故 torch 中对 dtype 是 非浮点数 的 dtype 没有实现，解决办法是对非浮点类型转换到 torch.float64，得到结果后转换回原来的 dtype。

```python
if isinstance(input, torch.Tensor):
    src_dtype = input.dtype
    if input.dtype not in [torch.float16, torch.float32, torch.float64]:
        input = input.to(torch.float64)
else:
    raise ValueError(f"input must be a tensor, but got {type(input)}")
result = torch.frac(input)
if src_dtype != result.dtype:
    result = result.to(src_dtype)
```

配置均通过，将 pending 中的移入 accuracy_8.txt

## paddle.nn.functional.thresholded_relu

有对应 torch api 且参数均同名，但是 torch 同名参数没有默认值，设置默认值

```json
"set_defaults": {
    "threshold": 1.0,
    "value": 0.0
}
```

配置均通过，将 pending 中的移入 accuracy_8.txt

### CODE REVIEW

File: tester/paddle_to_torch/mapping.json [hunk 1]
```diff
@@ -2053,6 +2067,22 @@
     "paddle.reverse":{
         "Rule":"ReverseRule"
     },
+    "paddle.fft.rfft2": {
+        "torch_api": "torch.fft.rfft2",
+        "set_defaults": {
+            "s": "None", 
+            "dim": "(-2, -1)", 
+            "norm": "None"
+        },
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/mapping.json:2076]: set_defaults 是设定的 paddle 参数的默认值，键为 paddle 的关键字

设定后，torch_kwargs 可以直接访问，不必再使用 locals().get('axes')

File: tester/paddle_to_torch/rules.py [hunk 2]
```diff
@@ -1537,6 +1537,27 @@ def apply(self, paddle_api: str) -> ConvertResult:
         code = Code(preprocess=pre.splitlines(), core=core.splitlines())
         return ConvertResult.success(paddle_api, code, is_torch_corresponding=False)
 
+
+class FracRule(BaseRule):
+    def apply(self, paddle_api: str) -> ConvertResult:
+        pre = """
+input = locals().get('x')
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/rules.py:1544]: x 参数一定会被提供，可以直接访问：
```suggestion
input = x
```

**Review by cangtianhuang — CHANGES_REQUESTED** [review]

File: tester/paddle_to_torch/rules.py (hunk 2 above)
Comment (Cutelemon6) [reply tester/paddle_to_torch/rules.py]: 已修改，那就直接重命名 input 就好了

**Review by cangtianhuang — APPROVED** [review]
LGTM

**Review by wanghuancoder — APPROVED** [review]
LGTM
